API/Supported_Languages.md

```diff
@@ -7,9 +7,10 @@ Tessel 2 has full support for JavaScript and Node.js. The relevant repos can be
 * [CLI support for JavaScript/Node](https://github.com/tessel/t2-cli/blob/master/lib/tessel/deployment/javascript.js)
 
 ### Binary modules
-There is support for binary modules on Tessel 2 that we have precompiled.
 
-The best way to find out whether the module you want is available is to try deploying it. If it's supported, it will work! If it is not supported, you will get an error message like this ([Error message source in CLI](https://github.com/tessel/t2-cli/blob/master/lib/tessel/deployment/javascript.js)):
+There is support for binary modules on Tessel 2 that are precompiled. The best way to find out whether the module you want is available is to try deploying it. The module has been precompiled, it will just work!
+
+For modules that have not been precompiled (you can see the list at [packages.tessel.io](packages.tessel.io)), you will see an error message like this ([Error message source in CLI](https://github.com/tessel/t2-cli/blob/master/lib/tessel/deployment/javascript.js)):
```

Comment:
I don't think it's appropriate to link to the source code and tell users to look for the error message. We can just include it here: 

```
Pre-compiled module is missing: NAME OF MODULE.
This might be caused by any of the following:
1. The binary is platform specific and cannot be compiled for OpenWRT.
2. A pre-compiled binary has not yet been generated for this module.
3. The binary didn't compile correctly for the platform that you're developing on.
    It's possible that the binary is Linux-only or even OpenWRT specific,
    try npm installing with "--force" and rerun your deployment command.
Please file an issue at https://github.com/tessel/t2-cli/issues/new
```